src/main/java/com/team871/subsystems/ArmExtension.java

```diff
@@ -1,31 +1,75 @@
 package com.team871.subsystems;
 
+import com.team871.config.DistanceEncoder;
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.util.sendable.SendableBuilder;
 import edu.wpi.first.wpilibj.motorcontrol.MotorController;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
 import edu.wpi.first.wpilibj2.command.CommandBase;
+import edu.wpi.first.wpilibj2.command.PIDCommand;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
 import java.util.function.Supplier;
 
 public class ArmExtension extends SubsystemBase {
 
+  private static final double EXTENSION_PID_KP = 0.00012207;
+  private static final double EXTENSION_PID_KI = 0;
+  private static final double EXTENSION_PID_KD = 0;
+
   private final MotorController extensionMotor;
+  private final PIDController extensionPID;
+  private final DistanceEncoder distanceEncoder;
+  private double positionInchesSetpoint;
```

Comment:
What does this add considering the PIDController on the dashboard and set point can be manipulated through that?